src/components/pages/index.js

```diff
@@ -1,19 +1,24 @@
+import { useState } from 'react';
 import { useRouter } from 'next/router';
 import MemberList from '@components/members-list';
 import classNames from '@components/pages/home-page.module.scss';
 import NewMemberList from '@components/new-member-list';
 import UserProfile from '@components/user-profile';
 import MemberRoleUpdate from '@components/member-role-update';
+import SearchMemberBox from '@components/search-members/index';
 import { userContext } from '@store/user/user-context';
 
 const HomePage = () => {
   const { showMemberRoleUpdateModal, isSuperUserMode } = userContext();
-
+  const [searchTerm, setSearchTerm] = useState('');
```

Comment:
move this to context and use it across the components where required.

Reply:
Done